airgun/views/computeresource.py

```diff
@@ -0,0 +1,39 @@
+from widgetastic.widget import Checkbox, Select, Text, TextInput
+from airgun.views.common import BaseLoggedInView, SearchableViewMixin
+from airgun.widgets import FilteredDropdown, ActionsDropdown
+
+
+class ComputeResourcesView(BaseLoggedInView, SearchableViewMixin):
+    title = Text("//h1[text()='Compute Resources']")
+    new = Text("//a[contains(@href, '/compute_resources/new')]")
+    edit = Text(".//span[contains(@class, 'btn')][a/text()='Edit']")
```

Comment:
usually we look for that element for edit:
![image](https://user-images.githubusercontent.com/10895065/39427207-9bdd5a4c-4c8b-11e8-8697-f990ec43badf.png)
If you still like to use that button - it is now a part of `ActionsDropdown` widget

Reply:
Right, `Edit` should be called via `action_list.fill('Edit')` and i'd rename it to just `actions.fill('Edit')`